todo_app/data/session_items.py

```diff
@@ -6,6 +7,8 @@
 from dateutil.parser import parse
 from todo_app.todo_item import TodoItem
 
+file_path = find_dotenv(".env")
+load_dotenv(file_path, override=True)
```

Comment:
I know explicitly suggested by me, but worth noting that this is potentially confusing behaviour for anyone else; there's no obvious sign that this would happen here, and hence no reason why this happens here
A simple comment explanation might make this a little easier, something like
```python
# When running in production, we need to explicitly load the environment variables before they're used in this file
```

Note that this will stop being true almost immediately, as we'll set up our environment variables slightly differently in Docker, but...

Reply:
Added :)